README.md

```diff
@@ -637,25 +637,25 @@ example that uses globals in a good way is a program that keeps a cache.
 Function names should be lowercase, with words separated by underscores as
 necessary to improve readability.
 
-!mixedCase is allowed only in contexts where that's already the prevailing
+mixedCase is allowed only in contexts where that's already the prevailing
 style (e.g. threading.py), to retain backwards compatibility.
 
```

Comment:
*'threading.py'* as example for mixedCase? :)  Maybe we can give a hint saying that this is the context and not the example.